libs/core/extend-expect.d.ts

```diff
@@ -32,3 +32,5 @@ declare module "brie" {
 }
 
 declare module "sqlite-parser"
+
+declare module "usb"
```

Comment:
its needed?

Reply:
No more. Forget to remove `usb-list.helper.ts` file, which used this solution.